src/axom/lumberjack/MPIUtility.cpp

```diff
@@ -22,23 +22,25 @@ namespace axom
 namespace lumberjack
 {
 
+const int LJ_TAG = 55432;
+
 const char* mpiBlockingReceiveMessages(MPI_Comm comm)
 {
   char* charArray = nullptr;
   int messageSize = -1;
   MPI_Status mpiStatus;
 
   // Get size and source of MPI message
-  MPI_Probe(MPI_ANY_SOURCE, 0, comm, &mpiStatus);
+  MPI_Probe(MPI_ANY_SOURCE, LJ_TAG, comm, &mpiStatus);
   MPI_Get_count(&mpiStatus, MPI_CHAR, &messageSize);
 
   // Setup where to receive the char array
   charArray = new char[messageSize+1];
   charArray[messageSize] = '\0';
 
   // Receive packed Message
-  MPI_Recv(charArray, messageSize, MPI_CHAR, mpiStatus.MPI_SOURCE, 0, comm,
-           &mpiStatus);
+  MPI_Recv(charArray, messageSize, MPI_CHAR, mpiStatus.MPI_SOURCE,
+           LJ_TAG, comm, &mpiStatus);
 
   return charArray;
```

Comment:
How is `charArray` deleted?

Reply:
It's deleted by the Lumberjack::pushMessagesOnce() or Lumberjack::pushMessagesFully()